CMakeLists.txt

```diff
@@ -38,9 +41,9 @@ if(EXISTS ${EXPAT_INCLUDE_DIRS})
   INCLUDE_DIRECTORIES(${EXPAT_INCLUDE_DIRS})
 endif(EXISTS ${EXPAT_INCLUDE_DIRS})
 
-if(EXISTS ${LIBXML2_INCLUDE_DIRS})
-  INCLUDE_DIRECTORIES(${LIBXML2_INCLUDE_DIRS})
-endif(EXISTS ${LIBXML2_INCLUDE_DIRS})
+if(EXISTS ${LIBXML2_INCLUDE_DIR})
```

Comment:
If this is changing to be `LIBXML2_INCLUDE_DIR` instead of `LIBXML2_INCLUDE__DIRS` why don't `LIBXSLT_INCLUDE_DIRS` etc. need renaming?